src/Security/Authorization/Common/src/Policy/CommonPolicyEvaluator.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Authorization.Policy
+{
+    public class CommonPolicyEvaluator : ICommonPolicyEvaluator
```

Comment:
Do we still need a PolicyEvaluator?  It was only introduced to tie authN + authZ together, I thought on the client side there's no authentication needed, so wouldn't the base `IAuthorizationService` be sufficient?

Reply:
Yes, we do want to evaluate policies in client-side code.

> I thought on the client side there's no authentication needed

That's not correct. Although authentication isn't *enforced* there (all true enforcement is server-side), we still want policies to exist as a way of informing the client about what the user is going to be allowed to do.